HotelApp/Models/QueryStringParameters.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace HotelApp.API.Models
+{
+    public abstract class QueryStringParameters
+    {
+        const int maxPageSize = 50;
```

Comment:
Max page size nema potrebe provjeravat. Ak korisnik želi dohvatit 10000 komada, zašto nebi smio? U pravilu ćeš na frontendu imat neke predefinirane veličine stranica, ali backend može podržat sve